### PR TITLE
test(guidepup): navigation

### PR DESCRIPTION
**Pull request by bruno-sch**
## Proposed changes

test: adds guidepup test for navigation

## Types of changes

-   [ ] Bugfix (non-breaking change which fixes an issue)
-   [x] New feature (non-breaking change which adds functionality)
-   [ ] Refactoring (fix on existing components or architectural decisions)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [ ] Documentation Update (if none of the other choices apply)

### CODE REVIEW

**Review by nmerget — DISMISSED** [review]

File: showcases/screen-reader/tests/navigation.spec.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+	testDefault({
+		test,
+		title: 'default',
+		description:
+			'should have texts inline or as data-label attributes (next())',
+		url: './#/05/navigation?page=density',
+		async testFn(voiceOver, nvda) {
+			if (nvda) {
+				await nvda?.next(); // Navigation Item 1
+				await nvda?.act(); // Open menu
```
Comment (mfranzke) [inline showcases/screen-reader/tests/navigation.spec.ts:15]:
```suggestion
				await nvda?.next(); // Navigation Item 2
				await nvda?.previous(); // Navigation Item 1
				await nvda?.act(); // Open menu
```

even though that NVDA and VoiceOver wouldn't be feature equal in this case, we would check whether the navigation is actually hidden from NVDA screenreader in the first place before interacting with the button.

Comment (nmerget) [reply showcases/screen-reader/tests/navigation.spec.ts:15]: Why do we need this? NVDA tells us with `..., menu button, collapsed,...` that the sub-navigation is hidden isn't it?

Comment (mfranzke) [reply showcases/screen-reader/tests/navigation.spec.ts:15]: It tells us that we've set an attribute that would announce `collapsed`, but it doesn't test for whether the navigation is actually visually hidden.

**Review by nmerget — DISMISSED** [review]

**Review by mfranzke — APPROVED** [review]

File: __snapshots__/tab-item/patternhub/tab-item-properties-should-match-screenshot.png
Comment (mfranzke) [inline __snapshots__/tab-item/patternhub/tab-item-properties-should-match-screenshot.png:1]: I think we do have a general problem here.

Comment (mfranzke) [reply __snapshots__/tab-item/patternhub/tab-item-properties-should-match-screenshot.png:1]: https://github.com/db-ui/mono/issues/3170

**Review by mfranzke — APPROVED** [review]

**Pull request merged**